### PR TITLE
add 1.8.0 notes to changelog

### PR DESCRIPTION
**Pull request by Cambaza**
- add 1.8.0 notes to changelog

### CODE REVIEW

**Review by fred-navruzov — APPROVED** [review]
LGTM with release date requested

**Review by dima-vm — APPROVED** [review]

File: docs/anomaly-detection/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Please find [launch instructions here](/vmanomaly.html#run-vmanomaly-docker-cont
 # tip
 
 
+## v1.8.0
+Released: 2024-01-15
+- FEATURE: Added Univariate [MAD (median absolute deviation)](https://en.wikipedia.org/wiki/Median_absolute_deviation) model support.
```
Comment (tenmozes) [inline docs/anomaly-detection/CHANGELOG.md:30]: plase provide link to the doc as well

File: docs/anomaly-detection/CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ Please find [launch instructions here](/vmanomaly.html#run-vmanomaly-docker-cont
...
+- IMPROVEMENT: Update Python to 3.12.1 and all the dependencies.
+- IMPROVEMENT: Don't check /health endpoint, check the real /query_range or /import endpoints directly. Users kept getting problems with /health.
```
Comment (tenmozes) [inline docs/anomaly-detection/CHANGELOG.md:32]: link to the issue if possible

File: docs/anomaly-detection/CHANGELOG.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ Please find [launch instructions here](/vmanomaly.html#run-vmanomaly-docker-cont
...
+- DEPRECATION: "health_path" param is deprecated and doesn't do anything in config ([reader](/anomaly-detection/components/reader.html#vm-reader), [writer](/anomaly-detection/components/writer.html#vm-writer), [monitoring.push](/anomaly-detection/components/monitoring.html#push-config-parameters)).
```
Comment (tenmozes) [inline docs/anomaly-detection/CHANGELOG.md:33]: should more visible check https://docs.victoriametrics.com/CHANGELOG.html#v1950

File: docs/anomaly-detection/CHANGELOG.md (hunk 2 above)
Comment (fred-navruzov) [reply docs/anomaly-detection/CHANGELOG.md:32]: we do not have direct (private) issues to link here, it's based on recent direct feedback from trials instead

File: docs/anomaly-detection/CHANGELOG.md (hunk 1 above)
Comment (fred-navruzov) [reply docs/anomaly-detection/CHANGELOG.md:30]: @Cambaza, we can just substitute this wiki link with the link to our model section - `/anomaly-detection/components/models.html# ...`, as we already have wiki link there

**Pull request merged**